spec/models/application_proceeding_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe ApplicationProceedingType, type: :model do
+  subject { described_class.new }
+
+  let!(:applicationProceedingType) { ApplicationProceedingType.create }
```

Comment:
please lets not Javify our Ruby code 😅 , the `let` variables should be defined like any normal ruby variable, not camelcase, but instead with underscores 👍 .